quantum/process_keycode/process_unicode_common.c

```diff
@@ -19,42 +19,46 @@
 #include <string.h>
 #include <ctype.h>
 
-static uint8_t input_mode;
-uint8_t mods;
+unicode_config_t unicode_config;
+static uint8_t saved_mods;
+#ifdef AUDIO_ENABLE
+  #ifdef UNICODE_LINUX_SONG
```

Comment:
```suggestion
  #ifdef UNICODE_SONG_LNX
```
I suggest renaming these so that they're consistent with `UNICODE_MODE_LNX` etc.; see above.